### ISSUE TITLE
[Debug] DebugClassLoader reports bad deprecation notice

### ISSUE THREAD

**Issue by alekitto** [symfony/symfony#29503]
**Symfony version(s) affected**: 4.2.1

**Description**
While using PHPUnit 7.5 mocks the DebugClassLoader reports
```
The "PHPUnit\Framework\MockObject\MockObject" interface extends "PHPUnit_Framework_MockObject_MockObject" that is deprecated Use PHPUnit\Framework\MockObject\MockObject instead.
```

As you can see, the first interface and one suggested are exactly the same that correctly extends the deprecated interface in order to not break BC.

**How to reproduce**  
Just use a simple `$this->createMock` in PHPUnit 7.5 test case.

**Comment by nicolas-grekas** [comment]
Should be fixed by #29587, please confirm if possible.

**Comment by alekitto** [comment]
Seems working correctly with #29587 applied
